Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -63,7 +51,17 @@ extension PhotonActionSheetProtocol {
             openURL(HomePanelType.readingList.localhostURL, false)
         }
         
-        return [openSettings, openTopSites, openBookmarks, openHistory, openReadingList]
+        let openHomePage = PhotonActionSheetItem(title: Strings.AppMenuOpenHomePageTitleString, iconString: "menu-Home") { _ in
+            guard let tab = self.tabManager.selectedTab else { return }
```

Comment:
nit: i think the style guide calls for this going on a new line (but i know we do this elsewhere too)

Reply:
We might want to state for our guidelines that single line guard with empty returns are ok.

I believe it is not explicit in our guidelines: https://www.raywenderlich.com/148830/swift-style-guide-updated-for-swift-3. 

```Formatting guard statements was also an item of some discussion. Should they be one-liners or always span multiple lines? How should you indent multiple let bindings and conditions? How should the else clause be indented?
In the end, we decided to not make a specific rule about it, and leave it to the discretion of the author. Perhaps a universal standard will emerge.```